### DISCUSSION TITLE
How to pause a process? And continue to execute

### DISCUSSION THREAD

**Discussion by zhanghaipeng666** [elsa-workflows/elsa-core#4997] [Q&A]

**Comment by sfmskywalker** [comment]
Elsa 3 currently doesn't have a way to pause a process, but is something we are planning to look into later this year.

**Comment by je-git-32** [comment]
I have a similar query regarding my workflow, which involves processing multiple chat data events. The basic concept is that the first message triggers the workflow effectively, and after processing, it's supposed to wait for further user input. It should enter a Suspended state upon encountering the next activity that processes chat data. My activity inherits from `Trigger<T>`, where `T` represents the specific type. Thus, the process is expected to pause until the next chat event occurs. However, this does not happen as intended; the initial event is passed to the described activity, and it doesn't wait for a subsequent event. Bookmarks have been appropriately set for all my activities. Is this behavior currently not feasible, or is there a flaw in my reasoning?
